basic-features.md

```diff
@@ -6,224 +6,359 @@ aliases: ['/docs/dev/basic-features/','/tidb/dev/experimental-features-4.0/']
 
 # TiDB Features
 
-This document lists the features supported in each TiDB version.
+This document lists the features supported in different TiDB versions, including [Long-Term Support (LTS)](/releases/versioning.md#long-term-support-releases) versions and [Development Milestone Release (DMR)](/releases/versioning.md#development-milestone-releases) versions after the latest LTS version.
 
 > **Note:**
 >
-> - Y: the feature is generally available (GA) and can be used in production environments.
+> PingCAP does not provide patch releases (including bug fixes and security vulnerability fixes) for DMR versions. If you encounter any bugs in a DMR version, they will be fixed in the subsequent releases. It is recommended to use the [latest LTS version](https://docs.pingcap.com/tidb/stable) unless you have special needs.
```

Comment:
Try to keep wording consistent with that in the **Important** module on https://docs.pingcap.com/tidb/v6.4.
```suggestion
> PingCAP does not provide patch releases for DMR versions. Any bugs will be fixed in future releases. For general purposes, it is recommended to use the [latest LTS version](https://docs.pingcap.com/tidb/stable).
```